nucliadb_dataset/nucliadb_dataset/dataset.py

```diff
@@ -468,7 +468,7 @@ def download_all_partitions(
     nucliadb_base_url: Optional[str] = "http://localhost:8080",
     path: Optional[str] = None,
     knowledgebox: Optional[KnowledgeBox] = None,
-    labels: List[str] = [],
```

Comment:
oof! this isn't good!

Reply:
I don't think we were hit by this yet, but just in case!